.github/workflows/test-suite-native-e2e-ios.yml

```diff
@@ -0,0 +1,149 @@
+
+      - name: Install Yarn dependencies
+        run: |
+          echo -e "\nenableScripts: false" >> .yarnrc.yml
+          echo -e "\nenableHardenedMode: false" >> .yarnrc.yml
+          yarn install
+
+      - name: Prebuild native expo project
+        working-directory: ./suite-native/app
+        run: yarn prebuild --platform ios --clean
+
+      - name: Sign message system config
+        working-directory: ./suite-common/message-system
+        run: yarn sign-config
+
+      - name: Use latest stable Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
+
+      - name: Build a Detox test app
+        working-directory: ./suite-native/app
+        run: $DETOX_BINARY_PATH build  --configuration ios.sim.release
+
+      - name: Save build to cache
+        uses: actions/cache/save@v4
+        with:
+          path: |
+            suite-native/app/ios/build/Build/Products/Release-iphonesimulator/TrezorSuiteLiteDebug.app
+          key: ios_test_build/${{ github.ref }}/${{github.run_id}}
+
+  run_ios_e2e_tests:
+    if: github.repository == 'trezor/trezor-suite'
+    runs-on: macos-latest
+    needs: prepare_ios_test_app
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v4
+        with:
+          submodules: "true"
+
+      - name: Install node and yarn
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+
+      - name: Load node modules cache
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: node_modules/${{ github.ref }}/${{github.run_id}}
+
+      - name: Install Yarn dependencies
+        run: |
+          echo -e "\nenableScripts: false" >> .yarnrc.yml
+          echo -e "\nenableHardenedMode: false" >> .yarnrc.yml
+          yarn install
+
+      - name: Install macOS dependencies
+        run: |
+          brew tap wix/brew
+          brew install wix/brew/applesimutils
+        env:
+          HOMEBREW_NO_AUTO_UPDATE: 1
+          HOMEBREW_NO_INSTALL_CLEANUP: 1
+
+      - name: Read test app build from cache
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            suite-native/app/ios/build/Build/Products/Release-iphonesimulator/TrezorSuiteLiteDebug.app
+          key: ios_test_build/${{ github.ref }}/${{github.run_id}}
+
+      - name: Prepare iPhone 11 Simulator
+        run: xcrun simctl create "iPhone 11"  com.apple.CoreSimulator.SimDeviceType.iPhone-11
+
+      - name: Run Detox E2E iOS tests
+        working-directory: ./suite-native/app
+        run: |
+          $DETOX_BINARY_PATH clean-framework-cache && $DETOX_BINARY_PATH build-framework-cache
+          yarn test:e2e ios.sim.release --headless --take-screenshots failing --record-videos failing
+
+      # TODO: add reporting to Currents.dev dashboard same as the android e2e tests have. cc @HajekOndrej
```

Comment:
Can we merge it after we fix this TODO, or is there some other stuff that needs to be done before we can do this?

Reply:
I have done the Currents TODO